tests/js/JSStandardBearTest.py

```diff
@@ -17,7 +17,7 @@
   bar = bar === 1
     ? bar
     : 1
-  if ((x = 33)) {
+  if ((x === 33)) {
```

Comment:
These changes to the tests should be a separate commit, which goes before your 'upgrade to eslint 3' commit, or even a separate PR.

If they are a separate PR we know that these test changes worked on ESLint 2 as well, before we upgrade to ESLint 3.  (Imagine that we need to revert your commit ... we would also revert these test changes, which isnt necessary.)

The extra commit / PR does not need a separate issue.  You can use `Related to https://github.com/coala/coala-bears/issues/872` for the last line of the commit.